deploy-docs.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+set -eu
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+
+cd "$GIT_ROOT"
+
+if [ -z "$(git status --porcelain)" ]
```

Comment:
The script doesn't get past this line for me, should it be `-n` instead of `-z` here?

Reply:
Damn, I switched away from `[[` because shellcheck dinged me, then I always get the tests wrong :(